src/Umbraco.Web.Common/Install/CreateUnattendedUserNotificationHandler.cs

```diff
@@ -0,0 +1,99 @@
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Identity;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Options;
+using Umbraco.Cms.Core.Configuration.Models;
+using Umbraco.Cms.Core.Events;
+using Umbraco.Cms.Core.Models.Membership;
+using Umbraco.Cms.Core.Security;
+using Umbraco.Cms.Core.Services;
+using Umbraco.Core.Events;
+using Umbraco.Extensions;
+
+namespace Umbraco.Cms.Web.Common.Install
+{
+    public class CreateUnattendedUserNotificationHandler : INotificationAsyncHandler<UnattendedInstallNotification>
+    {
+        private readonly GlobalSettings _globalSettings;
+        private readonly IUserService _userService;
+        private readonly IServiceScopeFactory _serviceScopeFactory;
+
+        public CreateUnattendedUserNotificationHandler(IOptions<GlobalSettings> globalSettings, IUserService userService, IServiceScopeFactory serviceScopeFactory)
+        {
+            _globalSettings = globalSettings.Value;
+            _userService = userService;
+            _serviceScopeFactory = serviceScopeFactory;
+        }
+
+        /// Listening for when the UnattendedInstallNotification fired after a sucessfulk
+        /// </summary>
+        /// <param name="notification"></param>
+        public async Task HandleAsync(UnattendedInstallNotification notification, CancellationToken cancellationToken)
+        {
+
+                // Ensure we have the setting enabled (Sanity check)
+                // In theory this should always be true as the event only fired when a sucessfull
+                if (_globalSettings.InstallUnattended == false)
+                {
+                    return;
+                }
+
+                var unattendedName = _globalSettings.UnattendedUserName;
+                var unattendedEmail = _globalSettings.UnattendedUserEmail;
+                var unattendedPassword = _globalSettings.UnattendedUserPassword;
+
+                // Missing configuration values (json, env variables etc)
+                if (unattendedName.IsNullOrWhiteSpace()
+                    || unattendedEmail.IsNullOrWhiteSpace()
+                    || unattendedPassword.IsNullOrWhiteSpace())
+                {
+                    return;
+                }
```

Comment:
This is okay, but I think you should consider implementing some validation logic in `GlobalSettingsValidator`..

Currently, I guess It will explode if I use a non-email as `UnattendedUserEmail`.

You could also failed earlier if some, but not all of these settings are set.